openwisp_controller/config/admin.py

```diff
@@ -600,9 +601,8 @@ def clone_selected_templates(self, request, queryset):
     actions = ['clone_selected_templates']
 
 
-if not app_settings.CONFIG_BACKEND_FIELD_SHOWN:  # pragma: nocover
+if app_settings.CONFIG_BACKEND_FIELD_SHOWN is False:  # pragma: nocover
```

Comment:
@nemesisdesign I have replaced `if not app_settings.CONFIG_BACKEND_FIELD_SHOWN:` with this as well. Should I revert this or is it good?